website/blog/2020-07-14-lithium.md

```diff
@@ -0,0 +1,116 @@
+---
+author: Tomasz Godzik
+title: Metals v0.9.2 - Lithium
+authorURL: https://twitter.com/TomekGodzik
+authorImageURL: https://github.com/tgodzik.png
+---
+
+We're happy to announce the release of Metals v0.9.2, which adds the support for
+Scala 2.12.12.
+
+<table>
+<tbody>
+  <tr>
+    <td>Commits since last release</td>
+    <td align="center">41</td>
+  </tr>
+  <tr>
+    <td>Merged PRs</td>
+    <td align="center">12</td>
+  </tr>
+    <tr>
+    <td>Contributors</td>
+    <td align="center">6</td>
+  </tr>
+  <tr>
+    <td>Closed issues</td>
+    <td align="center">14</td>
+  </tr>
+  <tr>
+    <td>New features</td>
+    <td align="center">1</td>
+  </tr>
+</tbody>
+</table>
+
+For full details: https://github.com/scalameta/metals/milestone/25?closed=1
+
+Metals is a language server for Scala that works with VS Code, Vim, Emacs,
+Sublime Text, Atom and Eclipse. Metals is developed at the
+[Scala Center](https://scala.epfl.ch/) and [VirtusLab](https://virtuslab.com)
+with the help from [Lunatech](https://lunatech.com) along with contributors from
+the community.
+
+## TL;DR
+
+Check out [https://scalameta.org/metals/](https://scalameta.org/metals/), and
+give Metals a try!
+
+- Scala 2.12.12 support
+
+## Miscellaneous improvements
+
+- Named parameters completions fixed to work in cases where the method was
+  defined later in the class.
+- Add option to not install and apply bloop in gradle builds.
```

Comment:
🤔 I'd maybe clarify this a bit since I'm not even 100% sure what it means.

> not install and apply bloop in gradle build

Does this mean gradle is the BSP server here that it's not using Bloop? I'd clarify what is meant.

Reply:
Changed to be explicit that we just don't add the additional plugin.